### PR TITLE
Doc enhancements

### PR DESCRIPTION
**Pull request by jlnbuiles**
Hey @nicklockwood. made a small change to the doc to specify the wrapping exclusion for comments and literals.

### CODE REVIEW

**Comment by nicklockwood** [comment]
Thanks! Although, this comment may soon be only party accurate 👀: https://github.com/nicklockwood/SwiftFormat/pull/1151

File: Rules.md [hunk 1]
```diff
@@ -170,7 +170,7 @@ swift version is set to 4.1 or above.
 
 ## assertionFailures
 
-Changes all instances of assert(false, ...) to assertionFailure(...) 
+Changes all instances of assert(false, ...) to assertionFailure(...)
```
Comment (nicklockwood) [inline Rules.md:173]: Was this an intentional change?

File: Rules.md [hunk 2]
```diff
@@ -1994,6 +1994,8 @@ Option | Description
 
 Wrap lines that exceed the specified maximum width.
 
+_note: wrapping is currently NOT supported for comments or string literals._
+
```
Comment (nicklockwood) [inline Rules.md:1998]: This text in this file is actually generated automatically from the Examples.swift file, so can you make the change there instead, then run the Metadata tests to update this file? Thanks.

Comment (nicklockwood) [reply Rules.md:1998]: Also, the styling here is different from the rest of the examples. I normally use:

```
**NOTE:** Blah blah
```

File: Rules.md (hunk 1 above)
Comment (jlnbuiles) [reply Rules.md:173]: hey nick. not at all. can remove it 👌

File: Rules.md (hunk 2 above)
Comment (jlnbuiles) [reply Rules.md:1998]: yes, i didn't know but will do that. probably by monday evening. sorry if didn't forsee that

**Pull request closed without merging**